app/src/main/res/layout/activity_main.xml

```diff
@@ -1,49 +1,135 @@
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_five" />
+
+            <Button
+                android:id="@+id/button7"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_six" />
+
+            <Button
+                android:id="@+id/button8"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_minus" />
+
+            <Button
+                android:id="@+id/button9"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_one" />
+
+            <Button
+                android:id="@+id/button10"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_two" />
+
+            <Button
+                android:id="@+id/button11"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_three" />
+
+            <Button
+                android:id="@+id/button12"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_multiplication" />
+
+            <Button
+                android:id="@+id/button13"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_zero" />
+
+            <Button
+                android:id="@+id/button14"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_split" />
+
+            <Button
+                android:id="@+id/button15"
+                style="@style/calculatorButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/main_screen_btn_text_erase" />
+        </GridLayout>
+    </LinearLayout>
 
-</LinearLayout>
\ No newline at end of file
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
why do you have a constraint layout if you are not using it?

Reply:
Done